documentation/asciidoc/accessories/display/legacy.adoc

```diff
@@ -1,6 +1,30 @@
 == Legacy Support
 
-NOTE: These instructions are for the original Raspberry Pi 1 Model A and B boards only. You can identify an original board as it is the only model with a 26-pin GPIO header, all other models have the now-standard 40-pin connector.
+[NOTE]
+====
```

Comment:
Entire block is too big. Can and should be condensed to essentials. Put the command inline in the text, only use a [source] block for the file contents.

Reply:
Think I shortened as much as I possibly can.